### PR TITLE
Expose is_linked on defendant which are linked with a MAAT id

### PR DESCRIPTION
**Pull request by archferns**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/CACP-284)
Add an attribute to allow the UI to see if a defendant is linked.

## Checklist

Before you ask people to review this PR:

- [x] Tests and linters should be passing
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: swagger/v1/defendant.json [hunk 1]
```diff
@@ -95,6 +95,9 @@
         "national_insurance_number": {
           "$ref": "#/definitions/nino"
         },
+        "is_linked": {
+          "$ref": "#/definitions/is_linked"
```
Comment (jsugarman) [inline swagger/v1/defendant.json]: where is the `#/definitions/is_linked` defined - can't find

File: swagger/v1/defendant.json [hunk 2]
```diff
@@ -156,6 +159,9 @@
     "national_insurance_number": {
       "$ref": "#/definitions/nino"
     },
+    "is_linked": {
```
Comment (jsugarman) [inline swagger/v1/defendant.json]: [ditto](https://github.com/ministryofjustice/laa-court-data-adaptor/pull/108/files#r397858930)

File: swagger/v1/defendant.json (hunk 1 above)
Comment (archferns) [reply swagger/v1/defendant.json]: https://github.com/ministryofjustice/laa-court-data-adaptor/pull/108/files#diff-b5b19d5c1fe96713cebbaa2c10c54917R8
it is defined as `is_linked` on the serializer, which aliases to from `Defendant#linked?`

Comment (archferns) [reply swagger/v1/defendant.json]: Added the correct reference.

**Pull request merged**